### DISCUSSION TITLE
estimating emissions from the investment

### DISCUSSION THREAD

**Discussion by hajarpb** [USEPA/supply-chain-factors#13] [Q&A] [answered]
Hello 
Could you please provide an example of using the EPA EEIO factors for estimating emissions from the investment category (scope 3 category)?
Thanks

**Comment by emehlhop** [comment, marked as answer]
Hi! Not one of the researchers here, but a frequent factor user :) 
Following GHG Protocol Scope 3 Technical Guidance from chapter 15 (https://ghgprotocol.org/scope-3-technical-calculation-guidance):
-Company A (Reporting Company) invests in Company B and has 10% equity in Company B. 
-Company B is a software company whose revenue was $1,000,000 in the reporting year. 
-Assuming reporting year is 2018 for simplicity.
1. Choose an appropriate emission factor -> Detail, Industry, SEF+MEF: 511200 Software Publishers. Data year 2016 in 2018 USD.
2. Using the GWP of your choice, convert the factor to kg CO2e/2018 USD (pretend this came out to 0.5 kg CO2e/2018 USD)
3. Company A emissions for Category 15 = Company B Revenue * % equity * emission factor
     $1,000,000 * 10% * 0.5 kg CO2e/2018 USD / 1000 (to convert to metric tons) = **50 metric tons CO2e**

**Reply by MoLi7** [reply]
Thanks @emehlhop ! This is exactly what we hope for. 😄
